collector/infiniband_linux.go

```diff
@@ -57,6 +57,7 @@ func NewInfiniBandCollector() (Collector, error) {
 		"link_error_recovery_total":                  "Number of times the link successfully recovered from an error state",
 		"multicast_packets_received_total":           "Number of multicast packets received (including errors)",
 		"multicast_packets_transmitted_total":        "Number of multicast packets transmitted (including errors)",
+		"physical_state_id":                          "physical state of the InfiniBand port",
```

Comment:
Please uppercase this to be in line with other descriptions. Ideally you'd come up with a better description since this doesn't tell me much (but optional).

Reply:
Capitalized.